### ISSUE TITLE
Feature request - Balance motors

### ISSUE THREAD

**Issue by air1kdf** [RaceFlightOne/RaceFlightOne#246]
I need to balance the motors.  I would like to be able to turn on a specific motor to a preset value (like half throttle), and get a reading from the accelerometer.  This way I can place a piece of vinyl tape on a specific place and see if it gets better or worse.  Once the spot is detected, increase the tape until it gets worse.  This will drastically reduce vibration on the whole quadcopter.

This could be a killer feature for Raceflight.

**Comment by RWooten68** [comment]
Yeah, this is one feature I always flash back to bf for. So I can tell what motors are noisy at what throttle position.

**Comment by proggod** [comment]
We can add that in next version
